app/scripts/formValidation.js

```diff
@@ -0,0 +1,29 @@
+const contact = document.querySelector('#contact');
+const fullnameErr = document.querySelector('#fullname-err');
+const emailErr = document.querySelector('#email-err');
+const msgErr = document.querySelector('#msg-err');
+
+contact.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const elements = e.target;
```

Comment:
- [ ] Good job working on validating the email field, but when I entered the correct email and all other required fields the form was expected to be submitted successfully and it's not happening. Kindly make sure that the form data is submitted when all fields have valid values. 👍🏽